packages/libs/error-stack/src/report.rs

```diff
@@ -26,6 +26,17 @@ use crate::{
 /// Attachments, and objects [`provide`]d by a [`Context`], are directly retrievable by calling
 /// [`request_ref()`] or [`request_value()`].
 ///
+/// ## Formatting
+///
+/// `Report` implements [`Display`] and [`Debug`]. When utilizing the [`Display`] implementation,
```

Comment:
Can we add an example for each? 

Maybe it'd be better to restructure it into 3 subheadings for each type of formatting

Reply:
Added the different outputs to the already provided example: https://github.com/hashintel/hash/pull/1597/commits/68d12d8c7e581355e6f7642e9711fa2eb1e84ad7